Nemt/nemt.h

```diff
@@ -0,0 +1,24 @@
+#ifndef nemt_h
+#define nemt_h
+#include <nemt.h>
+#include <SPI.h>  
+#include <Pixy.h>
+
+class nemt {
```

Comment:
@magic-sudo class names should begin with uppercase letter and be in "format" of `CapWords`, however filenames should be all lowercase. Please change `nemt` to `Nemt` class back again.